src/Tests/Microsoft.Diagnostics.Monitoring.Tool.UnitTests/Microsoft.Diagnostics.Monitoring.Tool.UnitTests.csproj

```diff
@@ -0,0 +1,16 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+    <PropertyGroup>
+      <TargetFrameworks>netcoreapp3.1</TargetFrameworks>
+    </PropertyGroup>
+
+    <ItemGroup>
+      <Compile Include="..\Microsoft.Diagnostics.Monitoring.Tool.FunctionalTests\Options\OptionsExtensions.cs" Link="Options\OptionsExtensions.cs" />
```

Comment:
Would it make sense to move OptionsExtensions to TestCommon?

Reply:
It might, but none of the options classes are in that project. I'd probably have to move the links to those classes to that project.

In the long term, I'm almost thinking that everything should be refactored into more granular libraries and then only reference via project reference is allowed, because this link of source files is starting to get difficult to juggle.

Reply:
If we do decide on this refactor I would do it in the main branch first.

Reply:
> If we do decide on this refactor I would do it in the main branch first.

https://github.com/dotnet/dotnet-monitor/pull/692